config/software/datadog-agent.rb

```diff
@@ -28,9 +28,6 @@
   mkdir  "#{install_dir}/agent/"
 
   # Agent code
-  mkdir "#{install_dir}/agent/checks.d"
-  copy 'checks.d', "#{install_dir}/agent/"
```

Comment:
just a note: this uses the `checks.d` directory that used to be in the `dd-agent` repo, which doesn't exist since we use `integrations-core`